teamB/eunsong/db/migrations/20221005064543_create_users_table.sql

```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+CREATE TABLE users (
+		id INT NOT NULL AUTO_INCREMENT,
+		name VARCHAR(50) NOT NULL,
+		email VARCHAR(200) NOT NULL,
+		profile_image VARCHAR(1000) NULL,
+		password VARCHAR(200) NOT NULL,
+		created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+		updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
+		PRIMARY KEY(id)
```

Comment:
마찬가지로 indentation을 잘 맞춰주세요-!
```suggestion
    id INT NOT NULL AUTO_INCREMENT,
    name VARCHAR(50) NOT NULL,
    email VARCHAR(200) NOT NULL,
    profile_image VARCHAR(1000) NULL,
    password VARCHAR(200) NOT NULL,
    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
    updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
    PRIMARY KEY(id)
```